service/node.go

```diff
@@ -420,6 +428,33 @@ func (s *service) nodeProbe(ctx context.Context) error {
 	return nil
 }
 
+func (s *service) approveSDC(opts Opts) error {
+
+	for _, systemID := range connectedSystemID {
+		system := s.systems[systemID]
+
+		//fetch SDC details
+		sdc, err := s.systems[systemID].FindSdc("SdcGUID", opts.SdcGUID)
+		if err != nil {
+			return status.Errorf(codes.FailedPrecondition, "%s", err)
+		}
+
+		//fetch the restrictedSdcMode
+		if system.System.RestrictedSdcModeEnabled && system.System.RestrictedSdcMode == "Guid" {
```

Comment:
We should log/return something if restricted mode is on, but the mode isn't supported by driver

Reply:
done